### PR TITLE
Feature/nep 103 make a scheduled job to create lot info

### PR DESCRIPTION
**Pull request by YuriiZaripa**

### CODE REVIEW

File: src/main/java/com/parsing/rest/ProzorroParserResource.java [hunk 1]
```diff
@@ -34,7 +34,7 @@ public void parsing() {
     @ResponseStatus(HttpStatus.FOUND)
     public void parsingPDF() {
         try {
-            schedulerParsingPDF.scheduled();
+            schedulerParsingPDF.parsLotResultAndLotPDFResult();
```
Comment (OleksandrOpanasiuk) [inline src/main/java/com/parsing/rest/ProzorroParserResource.java]: *parse

File: src/main/java/com/parsing/mockdata/mock_data_generator/ParticipantMockDataSource.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+    private final ParticipantRepository participantRepository;
+
+    public ParticipantMockDataSource(ParticipantRepository participantRepository) {
+        this.participantRepository = participantRepository;
+        this.participants = generate();
+    }
+
+    private List<Participant> generate() {
+        List<Participant> dataList = new ArrayList<>();
```
Comment (FedosovMax) [inline src/main/java/com/parsing/mockdata/mock_data_generator/ParticipantMockDataSource.java]: participans?

File: src/main/java/com/parsing/mockdata/mock_data_generator/ParticipantMockDataSource.java [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+        Set<String> uniqueValue = new HashSet<>();
```
Comment (FedosovMax) [inline src/main/java/com/parsing/mockdata/mock_data_generator/ParticipantMockDataSource.java]: values? Should be plural

File: src/main/java/com/parsing/rest/LotInfoResource.java [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/v1")
+@RequiredArgsConstructor
+public class LotInfoResource {
+
+    private final Scheduler scheduler;
+
+    @GetMapping("/lot-info")
```
Comment (FedosovMax) [inline src/main/java/com/parsing/rest/LotInfoResource.java]: /lot-infos

File: src/main/java/com/parsing/rest/ProzorroParserResource.java (hunk 1 above)
Comment (YuriiZaripa) [reply src/main/java/com/parsing/rest/ProzorroParserResource.java]: Done

File: src/main/java/com/parsing/mockdata/mock_data_generator/ParticipantMockDataSource.java (hunk 2 above)
Comment (YuriiZaripa) [reply src/main/java/com/parsing/mockdata/mock_data_generator/ParticipantMockDataSource.java]: Done

File: src/main/java/com/parsing/rest/LotInfoResource.java (hunk 4 above)
Comment (YuriiZaripa) [reply src/main/java/com/parsing/rest/LotInfoResource.java]: Done

File: src/main/java/com/parsing/mockdata/mock_data_generator/ParticipantMockDataSource.java (hunk 3 above)
Comment (YuriiZaripa) [reply src/main/java/com/parsing/mockdata/mock_data_generator/ParticipantMockDataSource.java]: Done

**Review by FedosovMax — APPROVED** [review]

**Review by OleksandrOpanasiuk — APPROVED** [review]

**Pull request merged**